File: 2d a 1d/conversion.js
let matriz2D = [[1,2,3],[4,5,6],[7,8,9]];
let matriz1D = [];

console.log("Matriz 2D original:");
matriz2D.forEach(fila => {
    console.log(fila);
    matriz1D.push(...fila);
});

console.log("Matriz 1D:");
console.log(matriz1D);

File: 2d a 1d/conversion.py
matriz2D = [[1,2,3],[4,5,6],[7,8,9]]
matriz1D = []

print("Matriz 2D original:")
for fila in matriz2D:
    print(fila)
    matriz1D.extend(fila)

print("\nMatriz 1D:")
print(matriz1D)

